### ISSUE TITLE
'specimen collection time measurement datum' versus 'date of specimen collection' 

### ISSUE THREAD

**Issue by cstoeckert** [biobanking/biobanking#12]
We have two terms in OBIB essentially covering the same data item:
'specimen collection time measurement datum' (http://purl.obolibrary.org/obo/OBI_0001619) imported from OBI and 'date of specimen collection' (http://purl.obolibrary.org/obo/OMIABIS_0001039) inherited from OMIABIS. Ideally we should only be using one.

**Comment by mbrochhausen** [comment]
Obviously, I am biased (as the developer at OMIABIS), but I remember creating the class despite the fact that the OBI class existed. To me, calling the report of a time, a measurement is misleading. There is not span of time, that is measured there. We just report the current time. Measurements of time always need to be closely linked to a time interval that is measured (just as the interval between point A and B on a piece of paper can be measured).
The following are examples of a time measurement: "The time it took a specimen to get from the operating room to tissue procurement", "the time a specimen remained in storage", etc.

**Comment by masciam** [comment]
I  agree with Mathias specimen collection time measurement  seems to be more for time interval  especially looking at  "is about some (process boundary and (starts some specimen collection process)" 
process boundary  and starts some specimen collection process 
It seems to be created for a linking a series of processes that start with the sample collection.

**Comment by zhengj2007** [comment]
I think date is a time point and time duration is an interval. There is no distinguish between time duration and date in IAO. Current we defined both date and time duration under IAO:time measurement datum'. The OBI 'specimen collection time measurement datum' is used as date. So, it has same meaning as OMIABIS:'date of specimen collection'. 

The logical axiom of OBI term is trying to say the time at starting point of specimen collection process which is a date. It is not intend to say an interval.

**Comment by mbrochhausen** [comment]
What is the argument for IAO being correct on representing it this way?
I am not debating that a specimen collection time is a data item, I just
don't understand what is measured when I affirm that it is now 6:53 pm
Central time, June 15, 2016.
